### DISCUSSION TITLE
Rename Project

### DISCUSSION THREAD

**Discussion by khanra17** [ReVanced/revanced-discussions#386] [General] [answered]
The Project is alrady going towards right direction.
A universal apk patcher (Like LuckyPatcher OSS). So calling it Re'Vanced' may be misleading.

**Comment by oSumAtrIX** [comment, marked as answer]
We are continuing the idea of ReVanced with the YouTube patches. It's also what the project is currently known for mainly.

**Reply by khanra17** [reply]
Twitter & Redit patches are already available
And why limit this great project to only youtube ?

**Reply by oSumAtrIX** [reply]
Check the patches repository for a list of available patches. This project is not limited to YouTube.

**Comment by Sculas** [comment]
What we know as "Vanced" is more or less a concept, in my opinion. It's "\<app> advanced, but without ads".
